### PR TITLE
Fix subgoal velocities issue

### PR DESCRIPTION
**Pull request by ChristophPetzoldt**
Fix: Split the goal_vels to the subgoal_velocities list with correct length.

### CODE REVIEW

**Comment by AlexanderFabisch** [comment]
Two remarks:

* Could you add a test case that fails in the previous version and will succeed with this patch?
* Are `get_n_params()`, `set_params()` and `get_params()` consistent?

**Comment by ChristophPetzoldt** [comment]
The previous versions throws an interrupt (SIGABRT) if dmp.step() is called. 
This can be tested by calling dmp_seq.trajectory().

**Comment by AlexanderFabisch** [comment]
> This can be tested by calling dmp_seq.trajectory().

Could you integrate that in the unit test?

File: bolero/representation/test/test_dmp_seq.py [hunk 1]
```diff
@@ -163,9 +163,18 @@ def test_trajectory_generation():
     assert_almost_equal(traj[100,0], subgoal, places=2)
 
 
+def test_subgoal_velocities():
+    test_task_dims = range(1,100,1)
+    for n_task_dims in test_task_dims:
+        dmp_seq, subgoals = create_dmp_seq(n_task_dims=n_task_dims)
+        dmp_seq.set_params(dmp_seq.get_params())
+        for i in range(len(subgoals)):
+            assert_equal(len(dmp_seq.get_subgoal_velocity(i)),n_task_dims)
```
Comment (AlexanderFabisch) [inline bolero/representation/test/test_dmp_seq.py]: Please add a space after ","

You can automatically check PEP8 compliance of a file with the command line tools pep8 or flake8 (can be installed with pip).

File: bolero/representation/test/test_dmp_seq.py [hunk 2]
```diff
@@ -163,9 +163,18 @@ def test_trajectory_generation():
     assert_almost_equal(traj[100,0], subgoal, places=2)
 
 
+def test_subgoal_velocities():
```
Comment (AlexanderFabisch) [inline bolero/representation/test/test_dmp_seq.py]: ... or maybe `test_set_params_with_subgoal_velocities`?

File: bolero/representation/test/test_dmp_seq.py [hunk 3]
```diff
@@ -163,9 +163,18 @@ def test_trajectory_generation():
...
+    test_task_dims = range(1,100,1)
```
Comment (AlexanderFabisch) [inline bolero/representation/test/test_dmp_seq.py]: space after ","

File: bolero/representation/test/test_dmp_seq.py (hunk 1 above)
Comment (ChristophPetzoldt) [reply bolero/representation/test/test_dmp_seq.py]: Thanks, installed the tool pycodestyle as the use of the pep8 tool will be removed in a future release:

1.7.1 (2017-10-22)
Changes:
- Prominently note via warning message that the tool is no longer released as pep8 and will only be fixed in the pycodestyle package

Comment (ChristophPetzoldt) [reply bolero/representation/test/test_dmp_seq.py]: see [d45cdbc](https://github.com/rock-learning/bolero/pull/44/commits/d45cdbc499cd458ac97f506f4a8303d02b9d452b)

File: bolero/representation/test/test_dmp_seq.py (hunk 2 above)
Comment (ChristophPetzoldt) [reply bolero/representation/test/test_dmp_seq.py]: see [d45cdbc](https://github.com/rock-learning/bolero/pull/44/commits/d45cdbc499cd458ac97f506f4a8303d02b9d452b)

File: bolero/representation/test/test_dmp_seq.py (hunk 3 above)
Comment (ChristophPetzoldt) [reply bolero/representation/test/test_dmp_seq.py]: see [d45cdbc](https://github.com/rock-learning/bolero/pull/44/commits/d45cdbc499cd458ac97f506f4a8303d02b9d452b)

**Comment by ChristophPetzoldt** [comment]
The issue in the previous version when calling dmp_seq.trajectory() is already tested by the assert-tests, so from my point-of-view, this branch can be merged. :+1:

**Pull request merged**